src/test/java/sorinfoca/Tests/Tema8/TestPage/TestPages.java

```diff
@@ -0,0 +1,52 @@
+package sorinfoca.Tests.Tema8.TestPage;
+
+import org.testng.annotations.Test;
+import sorinfoca.Tests.Tema8.pages.*;
+import static org.testng.Assert.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class TestPages extends BaseTest {
+
+    @Test(groups = {"desktop", "mobile"})
+    public void testBasicAjaxPage() {
+        BasicAjaxPage basicAjaxPage = new BasicAjaxPage(driver);
+        basicAjaxPage.goToPage();
+        basicAjaxPage.selectOption("Server");
+        basicAjaxPage.selectLanguage("Java");
+        basicAjaxPage.clickCodeItInButton();
+        assertEquals(basicAjaxPage.getSelectedOption(),"23");
+    }
+
+    @Test(groups = {"desktop", "mobile"})
+    public void JavaScriptRedirectPage() {
+        JavaScriptRedirectPage javascriptRedirectPage = new JavaScriptRedirectPage(driver);
```

Comment:
This test should have the following steps as per manual flow -> go to page, click on one of the buttons, wait for redirect, assert that you landed on a different page after redirect. Please add the missing steps -> wait for redirect, assert that you landed on a different page after redirect. Thanks!